tests/test_advanced_target_loop.py

```diff
@@ -0,0 +1,260 @@
+# This file is part of ts_scheduler
+#
+# Developed for the LSST Telescope and Site Systems.
```

Comment:
Please change `LSST` to `Vera Rubin Observatory` and check all other files as well.